business_requirement_deliverable/models/business.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+# © <YEAR(2015)>
+# <Elico Corp, contributor: Eric Caudal, Alex Duan, Xie XiaoPeng(S)>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class BusinessRequirement(models.Model):
+    _inherit = "business.requirement"
+
+    deliverable_lines = fields.One2many(
+        comodel_name='business.deliverable.line',
+        inverse_name='business_requirement_id',
+        string='Deliverable Lines',
+        copy=False,
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    estimated_time_total = fields.Float(
+        compute='_get_estimated_time_total',
+        string='Total Estimated Time',
+        store=True,
+    )
+    summary_estimation = fields.Boolean(
+        string='Summary Estimation?',
+        default=True,
+        readonly=True,
+        states={'draft': [('readonly', False)]},
+        help='True: I will review the business case here.',
+    )
+    summary_estimation_note = fields.Text(
+        'Summary Estimation Note',
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+
+    @api.one
+    @api.depends(
+        'deliverable_lines.estimated_time')
+    def _get_estimated_time_total(self):
+        time_total = sum(
+            line.estimated_time for line in self.deliverable_lines
```

Comment:
s/estimated_time/resource_time
function should be on resource line, not deliverable line.
reminder:
BR => Deliverable => Resource